### PR TITLE
documentation/S3C-4278_review_s3c-4144_text

### PR DESCRIPTION
**Pull request by wabernatScality**
Chasing S3C-4144 with a text review.

### CODE REVIEW

**Comment by wabernatScality** [comment]
At your leisure, gents. Please make sure I haven't broken anything in my attempts to tidy up.

**Review by miniscruff — APPROVED** [review]

File: docs/milestone-1.md [hunk 1]
```diff
@@ -2,75 +2,78 @@
...
-1. Click `Enable`. (This step will create a Vault Account in the background)
-1. A `Storage Tenant ID` should be generated on the UI which is also the associated Vault Account's Account ID.
-
-### OSIS integration with vCloud Director OSE:
+1. Log in as a vCloud Director Administrator on vCloud Director UI.
+1. Navigate to the `Resources` tab.
+1. Create a new `Organization` (vCloud Director tenant) with required fields.
+1. Navigate to the `Object Storage` tab.
+1. The `Tenants` pane is activated and lists all the vCloud Director tenants with an associated `Storage Tenant ID` (if any exist).
+1. Click the tenant name created in the `Tenants` pane.
```
Comment (saitharunthodupunuru) [inline docs/milestone-1.md:64]: What do you think this looks like? Please feel free to ignore this if it does not make sense :) 
```suggestion
1. Click the newly created tenant's name on the `Tenants` pane.
```

File: docs/milestone-1.md [hunk 2]
```diff
@@ -2,75 +2,78 @@
...
+1. Enable the toggle button beside the Tenant name.
+1. Click `Enable`. (This creates a Vault account in the background)
+1. A storage tenant ID, which is also the associated Vault Account's Account ID, is generated on the UI.
```
Comment (saitharunthodupunuru) [inline docs/milestone-1.md:67]: I will still highlight `Storage Tenant ID` as this is a label on UI.
```suggestion
1. `Storage Tenant ID`, which is also the associated Vault Account's Account ID, is generated on the UI.
```

**Review by saitharunthodupunuru — CHANGES_REQUESTED** [review]

File: docs/milestone-1.md (hunk 1 above)
Comment (wabernatScality) [reply docs/milestone-1.md:64]: I think it looks good!

File: docs/milestone-1.md (hunk 2 above)
Comment (wabernatScality) [reply docs/milestone-1.md:67]: Sounds reasonable to be.

**Review by saitharunthodupunuru — APPROVED** [review]
LGTM

**Review by lucieleonard — APPROVED** [review]

**Pull request merged**